### PR TITLE
Update RandomAttribute doc to add Guid support

### PR DESCRIPTION
**Pull request by Greybird**
Hi,

This is a proposal to update the `RandomAttribute` documentation to match with the feature added by https://github.com/nunit/nunit/pull/3951.

### CODE REVIEW

**Comment by SeanKilleen** [comment]
@Greybird thank you for the contribution! ❤️ 

@mikkelbu, I'm thinking let's add an informational message about it for now, and you can set a follow-up to update the docs if it's back-ported. That way we can get this contribution in without it having to sit for too long.

**Comment by Greybird** [comment]
Hi @SeanKilleen, I added a note about the minimum version being 4.0. 
Tell me if you feel wording / position / style should be updated!

File: docs/articles/nunit/writing-tests/attributes/random.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ In general, the forms that specify a minimum and maximum should be used on argum
 
 Note that there is no constructor taking decimal values for min and max. This is because .NET does not support use of decimal in an attribute constructor.
 
+> [!NOTE]
+> `Guid` support for `RandomAttribute` is possible only from NUnit version 4.0.
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/attributes/random.md:45]:
```suggestion
> `Guid` support for `RandomAttribute` is available from version 4.0 onwards.
```

**Comment by SeanKilleen** [comment]
@Greybird I made just a minor language change; I'll mark this for auto-merge once the build finishes.

Thanks again!

**Pull request merged**